server/src/main/java/org/candlepin/resource/ConsumerResource.java

```diff
@@ -1877,6 +1880,11 @@ public void deleteConsumer(@Verify(Consumer.class) String uuid) {
         }
 
         consumerRules.onConsumerDelete(toDelete);
+        ContentAccessCertificate cert = toDelete.getContentAccessCert();
+
+        if (cert != null) {
+            cacCurator.delete(cert);
+        }
```

Comment:
Let's use `ContentAccessManager.removeContentAccessCert()` here instead, since it already performs the necessary checks, and we don't have to add another dependency to this class (ContentAccessManager is already a dependency)

Reply:
Also, this is a good chance to add some unit tests for `ContentAccessManager.removeContentAccessCert()` itself, since I don't see any in `ContentAccessManagerTest.java`. It's a simple method, but we should strive to increase test coverage of all the code we touch

Reply:
Done. Added unit test for same.